### ISSUE TITLE
[Suggestion] Enable Auto Change Target for Soft Targeting

### ISSUE THREAD

**Issue by leafymonster** [UnknownX7/ReAction#45]
Hello! I discovered your plugin today and, first, I just want to say its absolutely lovely!

TL;DR at the top - I don't know if its possible, but I would love it if 'Enable Auto Change Target' also worked on soft targeting rather than only working on normal targeting, or maybe if there was an extra option to allow it to work for soft targeting.

I play on controller and occasionally I trigger soft self-targeting when using D-Pad Up or D-Pad Down skills because I'm bad. Corresponding keybinds for mouse and keyboard would be under System, -> "Move Cursor up(down)/Cycle up(down) through Party List" near the top of the list. Your plugin is already amazing, but if it could fix this soft targeting problem I have, it would be an absolute godsend.

**Comment by Clocktown** [comment]
Bump. Would certainly be awesome to have this!

By the way, here is a quick tip to potentially lessen your issue of self-soft-targeting. Used to happen to me all the time as well. In my case the issue was just that pressing the D-Pad would very quickly fire a repeat press even if I don't intend to actually hold the button down and that then causes me to select myself as i let go of a trigger. The way I fixed that is that I use the Steam Controller Support to modify my controller - where I changed the D-Pad to trigger as a "Start Press" action which avoids those repeat presses. For situations where I do want it repeated (like UI) I simply added a Long Press action that would fire repeatedly and allows me to control how long i have to hold the button before i get those repeated inputs.
